### PR TITLE
adapting lcogtweather to contain the new method to get data

### PR DESCRIPTION
**Pull request by herpichfr**
changing the lcogtweather

### CODE REVIEW

File: chimera_ctioenviroment/instruments/lcogtweather.py [hunk 1]
```diff
@@ -34,38 +34,35 @@ class LCOGTScrapper(object):
...
+        for key in urls.keys():
+            try:
+                self.results[key] = requests.get(urls[key]).json()[-1]
+            except:
+                self.results[key] = {u'TimeStamp': u'',
+                                     u'TimeStampMeasured': u'',
+                                     u'Value': None,
+                                     u'ValueString': u''}
+
+        return self.results
```
Comment (wschoenell) [inline chimera_ctioenviroment/instruments/lcogtweather.py]: why do you need result as `self.result`?

File: chimera_ctioenviroment/instruments/lcogtweather.py [hunk 2]
```diff
@@ -90,12 +87,13 @@ def _update(self, value):
...
-        if all([v in value for v in
-                ['Humidity', 'Pressure', 'Temperature', 'Transparency', 'Dew Point', 'Wind', 'OK to open']]):
+        if all([v in value.keys() for v in
+                ['humidity', 'pressure', 'temperature', 'sky_transparency', 'dew_point',
+                 'wind_speed', 'wind_direction']]):
             if not value.has_key('Interlock Reason'):
                 value['Interlock Reason'] = ''
             self._results = value
-            self._results['utctime'] = datetime.datetime.strptime(value['utctime'], '%Y-%m-%d %H:%M UTC')
+            self._results['utctime'] = datetime.datetime.strptime(value['temperature']['TimeStamp'].replace('/', '-'), '%Y-%m-%d %H:%M:%S')
```
Comment (wschoenell) [inline chimera_ctioenviroment/instruments/lcogtweather.py]: can't you change `'%Y-%m-%d %H:%M:%S'` to `'%Y/%m/%d %H:%M:%S'` to remove the `replace()`?

File: chimera_ctioenviroment/instruments/lcogtweather.py [hunk 3]
```diff
@@ -265,6 +234,9 @@ def getMetadata(self, request):
 
 
 if __name__ == '__main__':
+    # test = LCOGTScrapper()
```
Comment (wschoenell) [inline chimera_ctioenviroment/instruments/lcogtweather.py]: remove comment

**Review by wschoenell — CHANGES_REQUESTED** [review]

File: chimera_ctioenviroment/instruments/lcogtweather.py (hunk 1 above)
Comment (herpichfr) [reply chimera_ctioenviroment/instruments/lcogtweather.py]: I don't. Just a donkey programmer thing.

File: chimera_ctioenviroment/instruments/lcogtweather.py (hunk 2 above)
Comment (herpichfr) [reply chimera_ctioenviroment/instruments/lcogtweather.py]: Done

**Comment by herpichfr** [comment]
Done

**Pull request merged**